Topics/Tech_Stacks/htmx.md

```diff
@@ -0,0 +1,83 @@
+# HTMX Documentation
+## Introduction
+HTMX is a modern web development tool designed to enhance HTML with dynamic capabilities 
+often associated with complex JavaScript frameworks. 
+
+HTMX extends the core idea of HTML, trying to fully ultilize the power of HTML as a hypertext to achieve similar results as javascript frameworks without deviating
+as much from the original web schematics. This leads to efficent and light weight websites built to last a long time but at the cost of the more complex and advanced javascript features.
+HTMX allows developers to create rich and interactive web applications by writing HTML attributes for AJAX requests, CSS transitions, WebSockets,
+and Server Sent Events. This simplicity and power of HTMX opens up a new world for web development.
+
```

Comment:
Maybe add a section explaining why you might want to use HTMX as opposed to alternatives.

Reply:
Hey! This is a great suggestion and I think you'll be happy with what I added. Because I made these comparisons, it inspired me also to included a section detailing the limitations of HTMX for when JavaScript might be a better alternative